docs/sources/deploy-kubernetes/_index.md

```diff
@@ -52,64 +53,64 @@ Use a custom namespace so that you do not have to overwrite the default namespac
    helm repo update
    ```
 
-   > **Note:** The Helm chart at [https://grafana.github.io/helm-charts](https://grafana.github.io/helm-charts) is a publication of the source code at [**grafana/pyroscope**](https://github.com/grafana/pyroscope/tree/next/operations/phlare/helm/phlare).
+   > **Note:** The Helm chart at [https://grafana.github.io/helm-charts](https://grafana.github.io/helm-charts) is a publication of the source code at [**grafana/pyroscope**](https://github.com/grafana/pyroscope/tree/next/operations/pyroscope/helm/pyroscope).
 
-1. Install Grafana Phlare using the Helm chart using one of the following options:
+1. Install Pyroscope using the Helm chart using one of the following options:
 
-   - Option A: Install Grafana Phlare as single binary
+   - Option A: Install Pyroscope as single binary
 
    ```bash
-   helm -n phlare-test install phlare grafana/phlare
+   helm -n pyroscope-test install pyroscope grafana/pyroscope
```

Comment:
assuming this command wil not work (?)

Reply:
Yes we need to keep it at `grafana/phlare` until the helm chart is renamed